roles/bootstrap-os/tasks/bootstrap-opensuse.yml

```diff
@@ -5,3 +5,4 @@
     state: present
   with_items:
     - python-cryptography
+    - python-pip
```

Comment:
That is not necessary as httplib2 is installed via system packages on opensuse

Reply:
Changed. Thanks